database/migrations/2020_10_09_143603_create_steps_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateStepsTable extends Migration
```

Comment:
I was thinking to make this Step model reusable for other features, if need be. However, that might be a bit overkill actually.   Also, it would need another column to determine which feature it belongs to (eg. `feature`, `type`).